nestkernel/model.h

```diff
@@ -250,22 +232,17 @@ class Model
   /**
    * Memory for all nodes sorted by threads.
    */
-  std::vector< sli::pool > memory_;
+  std::vector< std::vector< Node* > > memory_;
 };
 
 
 inline Node*
 Model::allocate( thread t )
 {
```

Comment:
Are `allocate()` and `allocate_()` actually good names? Wouldn't `create` or `clone` be more readable?